### ISSUE TITLE
Include state and/or country in location description from API

### ISSUE THREAD

**Issue by coreymckrill** [Automattic/nearby-wordpress-events#14]
This suggests and change to the API itself rather than this plugin, but is relevant to what information is returned and displayed by the widget. When you type in a location in the field and search, you get back the name of the nearest city matching that location, along with events for it. However, a city name is often not enough context to know for sure where it's referring to, especially weeks or months after the search was actually performed. Since the `location.description` string that is returned by the API is not used for subsequent searches, and since the geo database we're using does include state and country information, I think it would be beneficial to either concatenate it for the description, or include separate city, state, and country key values in the API response. So instead of showing just "Vancouver", the widget could show something like "Vancouver, WA, USA".

**Comment by iandunn** [comment]
Yeah, that would be good. I remember trying that and giving up for some reason, but looking at it again, I don't see why we couldn't.

We don't always have states/regions, but we could just leave that out when it's not present.
